packages/extension-ui/src/Popup/Derive/index.tsx

```diff
@@ -36,12 +36,20 @@ function Derive({ isLocked }: Props): React.ReactElement<Props> {
   const [account, setAccount] = useState<null | PathState>(null);
   const [name, setName] = useState<string | null>(null);
   const [parentPassword, setParentPassword] = useState<string | null>(null);
+  const [step, setStep] = useState(1);
 
   const parentGenesis = useMemo(
     () => accounts.find((a) => a.address === parentAddress)?.genesisHash || null,
     [accounts, parentAddress]
   );
 
+  const parentIsExternal = useMemo(
+    () => accounts.find((a) => a.address === parentAddress)?.isExternal || null,
+    [accounts, parentAddress]
+  );
+
+  const isExternalResult = parentIsExternal || 'false';
```

Comment:
Won't simply:
```suggestion
  const parentIsExternal = useMemo(
    () => accounts.find((a) => a.address === parentAddress)?.isExternal || 'false',
    [accounts, parentAddress]
  );
```
achieve the same effect?